PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -364,7 +363,7 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
 #the94X miniAOD V2 had a bug in the scale and smearing for electrons in the E/p comb
 #therefore we redo it but but we need use a new name for the userFloat as we cant override existing userfloats
 #for technical reasons
-for modifier in run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,:
+for modifier in run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016:
```

Comment:
This is "overwriting" the block above, plus does not treat the run2_miniAOD_80XLegacy case. Did you want to write run2_miniAOD_80XLegacy in place of run2_nanoAOD_94X2016?

Reply:
yes, sorry, about this. Fixed. Sorry about these mistakes, just getting tired and distracted.